src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Controller.cs

```diff
@@ -23,20 +22,16 @@ internal partial class Controller :
         ICommandHandler<InvokeQuickInfoCommandArgs>
```

Comment:
sorry this will need to be updated after my commanding changes in https://github.com/dotnet/roslyn/pull/23769. Hopefully it will be pretty straightforward.

Reply:
Not a problem. I've been pulling this PR through changes for about a year now. :smile: